pkgs/pyhanko/src/pyhanko/pdf_utils/text.py

```diff
@@ -231,7 +236,7 @@ def render(self):
         command_stream.append(positioning.as_cm())
 
         command_stream += [
-            b'BT',
+            b'BT %g %g %g rg' % self.style.text_color,
```

Comment:
Same comment about RGB defaults also applies here. Also, let's keep graphics state manipulation outside the BT/ET.